microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -54,17 +59,78 @@ include::modules/microshift-olm-deploy-ops-spec-ns.adoc[leveloffset=+2]
 //additional resources for working with operators after deployment
 [role="_additional-resources"]
 .Additional resources
+OpenShift documentation::
 * link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/operators/administrator-tasks#olm-upgrading-operators[Updating installed Operators]
 * link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/operators/administrator-tasks#olm-deleting-operator-from-a-cluster-using-cli_olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster using the CLI]
 
-include::modules/microshift-olm-deploy-op-disconnected.adoc[leveloffset=+2]
+[id="microshift-olm-red-hat-ops-mirror_{context}"]
+== Using widely available Red Hat Operators and using mirror registries
+You can also filter and prune catalogs to get specific Operators and mirror them. You can also use Operators offline or embedded in {op-system-ostree-first} images. To read more details about how to configure your systems for mirroring, use the links in the following "Additional resources" section. If you are ready to deploy Operators from widely available Red Hat catalogs, mirror them, or to embed them in {op-system-ostree} images, start with the following section, "Using oc mirror to select catalogs and list Operators."
 
 //additional resources for deploying operators in disconnected and offline environments
 [role="_additional-resources"]
 .Additional resources
-* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/operators/administrator-tasks#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for more information.
-* link:https://docs.openshift.com/container-platform/{ocp-version}/installing/disconnected_install/installing-mirroring-disconnected.html#installing-mirroring-disconnected[Mirroring images for a disconnected installation using the oc-mirror plugin]
+OpenShift documentation::
```

Comment:
Darragh has WIP PR for the repo guidelines that might help here: https://github.com/openshift/openshift-docs/pull/73514

So, I think you could simplify the formatting by using something like this: `[Using Operator Lifecycle Manager on restricted networks] (OpenShift Container Platform documentation)`.

You wouldn't need to do that for the MS docs, but you can specify it for the RHEL for Edge and OCP references.